### PR TITLE
Fix/cln signing token

### PR DESCRIPTION
**Pull request by tobi-bams**

### CODE REVIEW

**Review by Evanfeenstra — COMMENTED** [review]
It should be 
```
const now = moment().unix()
const tsBytes = Buffer.from(now.toString(16), 'hex')
const bytesBase64 = buff.toString('base64')
const bytesUtf8 = Buffer.from(bytesBase64, 'utf8')
const sig = await LND.signBuffer(bytesUtf8, ownerPubkey)
```

**Comment by Evanfeenstra** [comment]
can u add to tribe create test... check that the owner_pubkey equals your own pubkey after the tribe is created

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -66,21 +66,22 @@ jobs:
           npm install && npm run build && docker build -t sphinxlightning/sphinx-relay .
       - name: Checkout stack
         run: |
-          git clone https://github.com/stakwork/sphinx-stack.git stack
+          git clone -b feat/cln-support https://github.com/stakwork/sphinx-stack.git stack
```
Comment (kevkevinpal) [inline .github/workflows/integration_test.yml:69]: just adding a comment here to remember to merge stack before merging this PR

File: .github/workflows/integration_test.yml [hunk 2]
```diff
@@ -66,21 +66,22 @@ jobs:
...
           chmod 777 -R ./proxy;
+          chmod 777 -R ./cln;
       - name: Check for NODES.json
         uses: nick-fields/retry@v2
         with:
           timeout_minutes: 10
           max_attempts: 3
           command: |
-            GITACTION_ENV=gitactionenv docker-compose -f ./stack/alts/proxy.yml --project-dir ./stack up -d;
+            GITACTION_ENV=gitactionenv docker-compose -f ./stack/alts/cln-proxy.yml --project-dir ./stack up -d;
```
Comment (kevkevinpal) [inline .github/workflows/integration_test.yml:84]: just a question

So is the current test environment 3 LND nodes and 1 proxy node connected to CLN?

Comment (Evanfeenstra) [reply .github/workflows/integration_test.yml:84]: yes. but for automated tests we should have both LND+Proxy and CLN+Proxy setups (there are two different compose files)

**Pull request merged**